src/gui/src/layoutViewer.cpp

```diff
@@ -33,13 +33,14 @@
 #include "layoutViewer.h"
 
 #include <QApplication>
-#include <QDebug>
 #include <QDateTime>
+#include <QDebug>
 #include <QFileDialog>
 #include <QFont>
 #include <QGridLayout>
 #include <QHBoxLayout>
 #include <QImageWriter>
+#include <QtGlobal>
```

Comment:
why is this needed?

Reply:
This is for the version check I used in the NoShaping ifdef.